frontend/language/src/nb.json

```
@@ -737,12 +737,20 @@
   "top_menu.texts": "Tekst",
   "top_menu.texts_old": "Tekst *",
   "ux_editor.add_map_layer": "Legg til kartlag",
+  "ux_editor.add_option": "Legg til alternativ",
+  "ux_editor.add_option_description_add": "Legg til beskrivelse",
+  "ux_editor.add_option_description_placeholder": "Ingen beskrivelse",
+  "ux_editor.add_option_label": "Tekst",
+  "ux_editor.add_option_label_add": "Legg til tekst",
+  "ux_editor.add_option_label_placeholder": "Tekst mangler",
+  "ux_editor.add_option_value": "Verdi",
+  "ux_editor.add_option_value_error_duplicate": "Verdien må være unik.",
```

Hva med omformulere denne til å eksplisitt fortelle hva som må være unik? :) 
```suggestion
  "ux_editor.add_option_value_error_duplicate": "Det finnes allerede en radio-knapp med denne verdien.",
```